src/Schema/Tables/AllergySymptom.sql

```diff
@@ -0,0 +1,9 @@
+﻿CREATE TABLE [dbo].[AllergySymptom]
+(
+	[AllergySymptomId] INT NOT NULL PRIMARY KEY IDENTITY, 
+    [PatientAllergyId] INT NOT NULL, 
+    [SymptomId] INT NOT NULL, 
+    [IsDeleted] BIT NOT NULL, 
+    CONSTRAINT [FK_AllergySymptom_To_PatientAllergy] FOREIGN KEY (PatientAllergyId) REFERENCES PatientAllergy(PatientAllergyId), 
```

Comment:
Add [] for columns and spaces between ():
FOREIGN KEY ( [PatientAllergyId] ) REFERENCES PatientAllergy( [PatientAllergyId] )